files/scripts/gbsyncd.sh

```diff
@@ -3,7 +3,11 @@
 . /usr/local/bin/syncd_common.sh
 
 function startplatform() {
-    :
+    # Add gbsyncd to FEATURE table, if not in. It did have same config as syncd.
+    if [ -z $($SONIC_DB_CLI CONFIG_DB HGET 'FEATURE|gbsyncd' state) ]; then
+        local CMD="local r=redis.call('DUMP', KEYS[1]); redis.call('RESTORE', KEYS[2], 0, r)"
+        $SONIC_DB_CLI CONFIG_DB EVAL "$CMD" 2 'FEATURE|syncd' 'FEATURE|gbsyncd'
```

Comment:
Hi @jimmyzhai , @abdosi ,
shouldn't sonic-db-cli here query the global feature list but not the namespace.
This code adds the feature in namespace but the global feature list does not have the gbsyncd entry.
This does not fix the container_checker requirement which looks into the global feature table.

Thanks

Reply:
Docker syncd/gbsyncd is per-namespace. Should we have syncd/gbsyncd entry at per-namespace feature table?

Reply:
Feature table entry is present both in global and namespace config db.  Idea is the entry value will always be same and consistent across global and State DB.

When user enable/disable feature state using cli hostcfgd update value both global and namespace specific db's.
And when we do show commands we get the db value from global DB's and display them.

@jimmyzhai I agree with @anamehra can we update global Feature Table also when we are are doing for namespace so that table is consistent ? 

Also with this change we will not need this  https://github.com/Azure/sonic-buildimage/pull/10834

cc @arlakshm